samples/scala-valueentity-customer-registry/scala-value-entity-customer-registry.sublime-project

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
could you remove all sublime files and add to gitignore

Reply:
ah crap, will do.